releases/release-1.32/README.md

```diff
@@ -0,0 +1,89 @@
+| **Begin Feature blog freeze**                                                          | Comms Lead                    | 02:00 UTC Wednesday 30th October 2024 / 19:00 PDT Tuesday 30th October 2024   | week 8  |                                                        |
+| **Preparing for Feature blogs review — Initiating outreach**                           | Comms                         | Monday 11th November 2024                                                     | week 10 |                                                        |
+| 1.32.0-beta.0 released                                                                 | Branch Manager                | Tuesday 12th November 2024                                                    | week 10 |                                                        |
+| Docs deadline — PRs ready for review                                                   | Docs Lead                     | Tuesday 12th November 2024                                                    | week 10 |                                                        |
+| Deprecations and Removals blog published                                               | Comms                         | Thursday 14th November 2024                                                   | week 10 |                                                        |
+| **Begin [Burndown]** (Monday, Wednesday, and Friday meetings)                          | Lead                          | Monday 18th November 2024                                                     | week 11 |                                                        |
+| Release Highlights deadline                                                                  | Comms                         | Tuesday 19th November 2024                                                    | week 11 |                                                        |
+| Start final draft of Release Notes                                                     | Release Notes Lead            | Tuesday 19th November2024                                                     | week 11 |                                                        |
+| **Begin [Code Freeze]**                                                                | Branch Manager                | 02:00 UTC Wednesday 20th November 2024 / 19:00 PDT Tuesday 19th November 2024 | week 11 |                                                        |
+| Release blog ready to review                                                           | Comms / Docs                  | 02:00 UTC Wednesday 20th November 2024 / 19:00 PDT Tuesday 19th November 2024 | week 11 |                                                        |
+| Feature blogs ready to review                                                          | Enhancement Owner / SIG Leads | Friday 22nd November 2024                                                     | week 11 |                                                        |
```

Comment:
Proposal to shift feature blogs ready to review on Monday 24th of November.

Rationale is: 
- Avoid overlap of the Release Blog and the feature blogs ready-to-review deadlines
- Begin the ready-to-review ramble with a new PR wrangler for docs. My goal here will be to work closely with that PR wrangler (maybe even setting up a short check, async or sync for the overall blog statuses)

Reply:
Monday 25th November is the next usable day, changed.